cfgov/v1/util/queries.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import ref
+
+from ..models.learn_page import AbstractFilterPage
+
+logger = logging.getLogger(__name__)
+
+
+class FilterableQuery(object):
+
+    def __init__(self, form, hostname, page=None):
+        self.form = form
+        self.hostname = hostname
+        self.page = page
+
+    def update_categories(self):
+        """ This is a (hopefully) temporary solution for dealing w/ the fact
+        that we show Blog and Reports as options for filtering, but
+        aren't categories themselves. Rather, they consist of sub-categories,
+        but since we aren't showing those sub-categories to the end user,
+        selecting the parent category is equivalent to all of them
+        getting checked. The mapping of Blog and Reports to their
+        respective categories exists in cfgov/v1/util/ref.py
+        """
+        categories = self.form.cleaned_data.get('categories', [])
+        if 'blog' in categories:
+            self.form.cleaned_data['categories'].remove('blog')
+            for x in ref.categories[1][1]:
```

Comment:
I moved this into a function inside `FilterableListForm`.  It's a good idea to update the line to "Research Report" but since that changes what we display, I think we should address that in a separate task/ PR.